package.json

```diff
@@ -97,10 +97,10 @@
     "@mdx-js/loader": "1.6.22",
     "@microsoft/api-extractor": "7.31.2",
     "@microsoft/api-extractor-model": "7.24.2",
-    "@microsoft/tsdoc": "0.14.1",
     "@microsoft/eslint-plugin-sdl": "0.1.9",
     "@microsoft/load-themed-styles": "1.10.26",
     "@microsoft/loader-load-themed-styles": "2.0.17",
+    "@microsoft/tsdoc": "0.14.1",
```

Comment:
nit: do we need this change in this PR?

Reply:
It's something automated for sorting package entries. Don't see a reason to revert it 🙂